### PR TITLE
Toggle switch children

### PR DESCRIPTION
**Pull request by alandardic**
Hi @lbelinsk,

I have added children property to ToggleSwitch + tests.

Please review 

Thanks

### CODE REVIEW

File: packages/wix-ui-core/src/components/ToggleSwitch/ToggleSwitch.driver.ts [hunk 1]
```diff
@@ -24,6 +25,6 @@ export const toggleSwitchDriverFactory = ({element, componentInstance, eventTrig
         className: 'root label',
         property: 'border-radius'
       });
-    }
+    },
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/ToggleSwitch/ToggleSwitch.driver.ts]: remove comma

File: packages/wix-ui-core/src/components/ToggleSwitch/ToggleSwitch.protractor.driver.ts [hunk 2]
```diff
@@ -2,5 +2,6 @@ export const toggleSwitchDriverFactory = component => ({
   element: () => component,
   click: () => component.click(),
   isChecked: () => component.$('input').isSelected(),
-  isDisabled: () => !component.$('input').isEnabled()
+  isDisabled: () => !component.$('input').isEnabled(),
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/ToggleSwitch/ToggleSwitch.protractor.driver.ts]: Good work with the tests.
Now what we still need is docs.
Will call you guys after the review

File: packages/wix-ui-core/stories/index.tsx [hunk 3]
```diff
@@ -19,7 +19,7 @@ storiesOf('Components', module)
     <Input dataHook="story-input" />
   ))
   .add('ToggleSwitch', () => (
-    <ToggleSwitch/>
+    <ToggleSwitch dataHook="story-toggle-switch">Hello<i>?</i></ToggleSwitch>
```
Comment (lbelinsk) [inline packages/wix-ui-core/stories/index.tsx]: why the question mark? hello is enough, no ? :)

**Comment by alandardic** [comment]
Hi @lbelinsk,

I have fixed all the pr comments and added the comments you have asked (for Badge and ToggleSwitch drivers).

Please review and merge with master (if it's ready)

Thanks

File: packages/wix-ui-core/src/components/Badge/Badge.driver.ts [hunk 4]
```diff
@@ -8,17 +8,29 @@ export const badgeDriverFactory = ({element, componentInstance, eventTrigger}) =
...
 
   return {
+    /** exists method - checks if element exists */
     exists: () => !!element,
+    /** getType method - returns elements type */
     getType: () => element.getAttribute('type'),
+    /** getType method - returns elements innerHTML */
     getContent: () => element.innerHTML,
+    /** getHeight method - returns elements height css property*/
     getHeight: () => domTestkit.getCssValue({className: 'badge', property: 'height'}),
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Badge/Badge.driver.ts]: Don't kill me, but I had an idea maybe to have another layer for the "style" methods.
```
return {
  exists: () => !!element,
  exists: () => !!element,
  getContent: () => element.innerHTML,
  styles: {
      getHeight: () => domTestkit.getCssValue(...,
      getPadding: () => domTestkit.getCssValue(...,
      ....
   }
}
```

**Comment by lbelinsk** [comment]
Guys, so much mixing between pr's (Badge & Toggleswitch are mixed)

File: packages/wix-ui-core/src/components/ToggleSwitch/ToggleSwitch.e2e.ts [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  const storyUrl = getStoryUrl('Components', 'ToggleSwitch');
+
+  beforeEach(() => browser.get(storyUrl));
+
+  eyes.it('should display correct content', () => {
+    const dataHook = 'story-toggle-switch';
+    const driver = toggleSwitchTestkitFactory({dataHook});
+
+    return waitForVisibilityOf(driver.element(), 'Cannot find ToggleSwitch')
+      .then(() => expect(driver.getTextContent()).toBe('Hello'));
```
[the remark this answers is not in the document]
Comment (argshook) [reply packages/wix-ui-core/src/components/ToggleSwitch/ToggleSwitch.e2e.ts]: e2e infrastructure ready === autotools ready
unfortunately above currently evaluates to `false`

to make that `true` autotools needs to:
1. better understand typescript
1. resolve components that are wrapped in HOCs (like `withClasses`)

File: packages/wix-ui-core/src/components/Badge/Badge.driver.ts [hunk 6]
```diff
@@ -8,17 +8,29 @@ export const badgeDriverFactory = ({element, componentInstance, eventTrigger}) =
   }
 
   return {
+    /** exists method - checks if element exists */
     exists: () => !!element,
+    /** getType method - returns elements type */
```
Comment (argshook) [inline packages/wix-ui-core/src/components/Badge/Badge.driver.ts]: @alandardic hey, no need to label each, autodocs (when they work ;) ) will use actual method name. so just description is enough

File: packages/wix-ui-core/src/components/ToggleSwitch/ToggleSwitch.protractor.driver.ts [hunk 7]
```diff
@@ -1,6 +1,12 @@
 export const toggleSwitchDriverFactory = component => ({
+  /** element method - returns the component element */
```
[the remark this answers is not in the document]
Comment (lbelinsk) [reply packages/wix-ui-core/src/components/ToggleSwitch/ToggleSwitch.protractor.driver.ts]: @argshook already commented about it here :)

File: packages/wix-ui-core/src/components/ToggleSwitch/ToggleSwitch.e2e.ts (hunk 5 above)
[the remark this answers is not in the document]
Comment (alandardic) [reply packages/wix-ui-core/src/components/ToggleSwitch/ToggleSwitch.e2e.ts]: Hi @lbelinsk,

So, currently I can't check toggle checked/ unchecked functionality - using e2e, right?

File: packages/wix-ui-core/src/components/ToggleSwitch/theme.ts [hunk 8]
```diff
@@ -1,35 +1,38 @@
...
+  colorChecked: string;
+  colorDisabled: string;
+  colorCheckedDisabled: string;
+  colorHover: string;
+
+  toggleIconWidth: string;
+  toggleIconHeight: string;
+  toggleIconDisplay: string;
+
+  childrenColor: string;
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/ToggleSwitch/theme.ts]: childrenColor is too low level, and talks about implementation.
What ablit LabelColor?

**Pull request merged**